ddtrace/tracer/tracer.go

```diff
@@ -401,6 +407,12 @@ func (t *tracer) worker(tick <-chan time.Time) {
 			}
 			return
 		}
+
+		t.totalTracesDropped.mu.Lock()
+		if t.totalTracesDropped.count > 0 {
+			log.Error("%d traces dropped through payload queue", t.totalTracesDropped.count)
+		}
+		t.totalTracesDropped.mu.Unlock()
```

Comment:
Lines 411-415 will be called very often, and it's only reading the value (as opposed to writing/modifying). So I'm wondering if we want to give `totalTracesDropped` a RWMutex, instead of a regular Mutex.
@darccio , any thoughts on this?

Reply:
@mtoffl01 That's a good suggestion. I'd do it, it makes sense.